test/yarn-auditer.js

```diff
@@ -27,6 +29,14 @@ function testDir(s) {
   return path.resolve(__dirname, s);
 }
 
+function canRunYarnBerry() {
+  const nodeVersion = childProcess
+    .execSync("node -v")
+    .toString()
+    .replace("\n", "");
+  return semver.gte(nodeVersion, "12.13.0");
```

Comment:
Do you think we need an equivalent check in the runtime for `audit-ci`? I suspect that it's redundant, just bouncing off ideas!

Reply:
I had a bit of a think about this. While we could do something like what is currently done with `isYarnVersionSupported` and throw an error message if the node version is not supported, I feel like this puts more maintenance overhead on audit-ci. Consumers will all be using different versions of yarn, and future major versions of yarn will likely deprecate older node runtimes. So if audit-ci were to perform its own checks, it would need a mapping of yarn versions to node version requirements (e.g. `yarn v2 requires node >=12`, `yarn v3 requires node >=14`).

I also think it would be unlikely for consumers to come across this error in audit-ci as if they are already using yarn they would be aware of its specific node version requirements.

Reply:
I fully agree with your reasoning. Great job!